src/java/org/apache/cassandra/index/sai/QueryContext.java

```diff
@@ -57,11 +72,21 @@ public long totalQueryTimeNs()
         return Clock.Global.nanoTime() - queryStartTimeNanos;
     }
 
+    public void incSstablesHit()
```

Comment:
Most counters on this class are directly incremented, do we need a separate method only for this one?